HeadlinesTests/RecentNewsDataSourceTests.swift

```diff
@@ -16,15 +16,17 @@ class RecentNewsDataSourceTests: XCTestCase {
     func testFetchNews() {
         let exp = expectation(description: "Get fake response from JSON file")
 
-        let success: ([News]) -> Void = { (_) in
-            exp.fulfill()
+        let handler: ((Result<[News], Error>) -> Void) = { result in
+            switch result {
+            case .success(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/cafeb7b8636cbafd0216eaa2489bb108aa52959c/HeadlinesTests/RecentNewsDataSourceTests.swift#L21"></span>Arguments can be omitted when matching enums with associated types if they are not used.<br />
<code>empty_enum_arguments</code> <code>RecentNewsDataSourceTests.swift:21</code></td>
    </tr>
  </tbody>
</table>